Demo/Measures/AdultOutpatientEncountersFHIR4-2.2.000.cs

```diff
@@ -133,8 +133,8 @@ private IEnumerable<Encounter> Qualifying_Encounters_Value()
 		var n_ = context.Operators.ListUnion<Encounter>(k_, m_);
 		bool? o_(Encounter ValidEncounter)
 		{
-			var q_ = context.Operators.Convert<string>(ValidEncounter?.StatusElement);
-			var r_ = context.Operators.Equal(q_, "finished");
+			var q_ = FHIRHelpers_4_0_001.ToString(ValidEncounter?.StatusElement);
```

Comment:
I think this should be the original Convert<string>, but that's because of a piece of code that we removed and that I commented on above.